### PR TITLE
Add pdblp recipe

### PR DESCRIPTION
**Pull request by matthewgilbert**
@conda-forge/help-python

### CODE REVIEW

**Comment by matthewgilbert** [comment]
@conda-forge/staged-recipes just following up on this, let me know if there is something that needs to be fixed for this to be merged.

File: recipes/pdblp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/pdblp/pdblp-{{ version }}.tar.gz
+  sha256: ccbecf941f1e88b85c6d6801785e0f76a54280dc3d2d39e7bd3836457da57192
+
+build:
+  number: 0
+  string: py{{ CONDA_PY }}_{{ PKG_BUILDNUM }}
+  script: python -m pip install --no-deps --ignore-installed .
```
Comment (scopatz) [inline recipes/pdblp/meta.yaml]: Can this be `noarch: python`?

Comment (matthewgilbert) [reply recipes/pdblp/meta.yaml]: Good point, will add

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**